rpc/handlers.go

```diff
@@ -1067,15 +1067,15 @@ func (h *Handler) AddTransaction(txnJSON json.RawMessage) (*AddTxResponse, *json
 		return nil, jsonrpc.Err(jsonrpc.InvalidJSON, err.Error())
 	}
 
-	if txnType, typeFound := request["type"]; typeFound && txnType == TxnInvoke.String() {
+	if txnType := request["type"]; txnType == TxnInvoke.String() {
```

Comment:
I also noticed that INVOKE_TXN_V0 also requires "contract_class": https://github.com/starkware-libs/starknet-specs/blob/v0.6.0-rc3/api/starknet_api_openrpc.json#L2482

Reply:
I'm also wondering, why part of validation we have here, and other here? 
https://github.com/NethermindEth/juno/blob/9448c9f56c33196c8aacc1d86c3cd23d3d7b424b/rpc/transaction.go#L215

Reply:
That is `contract_address`